docs/src/index.md

```diff
@@ -11,6 +11,10 @@ Aqua.jl provides functions to run a few automatable checks for Julia packages:
 * There are no "obvious" type piracies.
 * The package does not create any persistent Tasks that might block precompilation of dependencies.
 
+```@docs
+Aqua
```

Comment:
As already pointed out, this is indeed a bit of a useless docstring to add. If typically packages do not have a docstring, which I think they indeed do not, it will hamper adoption of this feature if we impose it. Worse still, it might hurt Aqua as a whole if this test becomes a default or limit the chances of this test becoming a default.

Reply:
Yes, I'll remove that requirement.